tests/fuzz/corpus/s2n_recv_client_supported_groups_test/1dc3882d4bcccb325751803b817489c3715db4cc

```diff
@@ -0,0 +1 @@
+�
```

Comment:
Is the missing newline intentional

Reply:
It's part of the auto-generated corpus so it should be.